controllers/volsync/volsync_suite_test.go

```diff
@@ -74,7 +76,8 @@ var _ = BeforeSuite(func() {
 
 	By("Setting up KUBEBUILDER_ASSETS for envtest")
 	if _, set := os.LookupEnv("KUBEBUILDER_ASSETS"); !set {
-		Expect(os.Setenv("KUBEBUILDER_ASSETS", "../../testbin/k8s/1.25.0-linux-amd64")).To(Succeed())
+		Expect(os.Setenv("KUBEBUILDER_ASSETS",
+			fmt.Sprintf("../../testbin/k8s/1.25.0-%s-%s", runtime.GOOS, runtime.GOARCH))).To(Succeed())
```

Comment:
Which code is responsible for downloading `../../testbin/k8s/1.25.0-linux-amd64`?

Can we get the name of the file from that code?

Reply:
Probably we can. I didn't want to bother with it. If you found out how to let me know.

Reply:
For the code that downloads the file is "make envtest"
```
KUBEBUILDER_ASSETS="$(shell $(ENVTEST_ASSETS_DIR)/setup-envtest use $(ENVTEST_K8S_VERSION) --bin-dir $(ENVTEST_ASSETS_DIR) -p path)"
```

Reply:
```
$ testbin/setup-envtest use --print path
/home/nsoffer/.local/share/kubebuilder-envtest/k8s/1.26.1-linux-amd64
```

Reply:
Right, but what are you trying to get to in order to help this hapless PR?

Reply:
Yeah, didn't want to spend too much time on this. Right now, I want my machine to be able to run unit tests.
Thanks for reviewing @nirs

Reply:
@nirs (just adding some context) The `make test` is always run by setting the KUBEBUILDER_ASSETS from the Makefile, the change to add this directly within the code was to reuse environments where we wanted to run test outside the make commands. This combined with the makefile based tests defaulting to verbose logging, even I end up running `go test ...` rather than use the `make test` directive most often. At which point the current change is required for non-linux setups.